[13.0][FIX] account_reconcile_payment_order: Fix tests

joao-p-marques opened the pull request:
Create journal instead of searching
Use computed value for total price instead of hardcoded (helps avoid integration tests when another module changes fiscal plan)

@Tecnativa
TT28962

ping @victoralmau @pedrobaeza

pedrobaeza commented on account_reconcile_payment_order/tests/test_account_reconcile_payment_order.py:
```diff
@@ -31,17 +25,29 @@ def setUpClass(cls):
+                        {
+                            "date": "2019-01-01",
+                            "name": "Test line",
+                            "amount": payment_order.total_company_currency,
```
This goes against the nature of the tests. We should fix data.

joao-p-marques replied:
Ok, I think it should be good now, along with https://github.com/OCA/bank-payment/pull/797 (this tests use the base from that module)

victoralmau replied:
PR it's complete now, it's ready to merge?

joao-p-marques replied:
It should be completed with the latest change, although test here will only pass after https://github.com/OCA/bank-payment/pull/798 is merged

joao-p-marques committed:
[FIX] account_reconcile_payment_order: Fix tests

Create journal instead of searching
Change hardcoded value for amount of payment statements, after we cleared the taxes of the invoice lines in https://github.com/OCA/bank-payment/pull/798

TT28962

pedrobaeza commented:
/ocabot merge patch
